pkgs/games/heroic/fhsenv.nix

```diff
@@ -40,6 +41,7 @@ buildFHSEnv {
       xorg.xrandr
       zstd
     ]
+    ++ lib.optional useNixpkgsUmu pkgs.umu-launcher
```

Comment:
I don't think it makes sense for this to be driven by a derivation flag.  If someone wants to use the upstream flake, they should apply it as an overlay to their NixOS or home-manager config, and then this reference to `umu-launcher` will point to the upstream version.

Reply:
I agree with this

Reply:
Understood! Thanks.